db/data_schema.rb

```diff
@@ -1 +1 @@
-DataMigrate::Data.define(version: 20250121164452)
+DataMigrate::Data.define()
```

Comment:
Can you revert this? I don't know why it's started making the change, I always have to remeber to ignore it and not commit it when there's a database update